core/services/aws/package.json

```diff
@@ -10,13 +10,16 @@
     "build": "lambdas=$(npm run -s find-lambdas --root_path=${npm_config_name} --filetype=ts) && esbuild $lambdas --bundle --minify --sourcemap --platform=node --target=node20 --outdir=${npm_config_out_dir}",
     "build-all": "npm run clear-build-dir && npm run build --name=** --out_dir=.build",
     "build-service": "npm run clear-build-dir && npm run build --out_dir=.build/${npm_config_name}/lambdas",
-    "recreate-zip-dir": "rm -rf .build/zips && mkdir .build/zips", 
-    "package":"lambdaJs=$(npm run -s find-lambdas --root_path=.build/${npm_config_name} --filetype=js) && npm run recreate-zip-dir && for file in $lambdaJs; do zip -j \"${file%.*}.zip\" \"$file\"; mv \"${file%.*}.zip\" .build/zips/; done",
-    "package-all": "npm run build-all && npm run package --name=**", 
+    "recreate-zip-dir": "rm -rf .build/zips && mkdir .build/zips",
+    "package": "lambdaJs=$(npm run -s find-lambdas --root_path=.build/${npm_config_name} --filetype=js) && npm run recreate-zip-dir && for file in $lambdaJs; do zip -j \"${file%.*}.zip\" \"$file\"; mv \"${file%.*}.zip\" .build/zips/; done",
+    "package-all": "npm run build-all && npm run package --name=**",
     "package-service": "npm run build-service && npm run package"
   },
   "dependencies": {
-    "aws-lambda": "^1.0.7"
+    "@aws-sdk/client-dynamodb": "^3.648.0",
+    "@aws-sdk/lib-dynamodb": "^3.648.0",
+    "aws-lambda": "^1.0.7",
+    "ksuid": "^3.0.0"
```

Comment:
Is it a stable library to use?

Reply:
Now using `ulid` as far as my findings I think it's more stable than ksuid. [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/112ef95c1f039771ad12885dad2dee375c65fe0c)